npm/vite-dev-server/client/initCypressTests.js

```diff
@@ -25,7 +61,7 @@ if (supportFile) {
   // We need a slash before /cypress/supportFile.js, this happens by default
   // with the current string replacement logic.
   importsToLoad.push({
-    load: () => import(`${devServerPublicPathRoute}${supportRelativeToProjectRoot}`),
+    load: () => importWithRetry(() => import(`${devServerPublicPathRoute}${supportRelativeToProjectRoot}`)),
```

Comment:
Could we refactor `importWithRetry` to accept the string path we want to import and just do the `import(..)` in there and repeat as needed? Then we wouldn't have to parse out the URL from the error message in that function